### PR TITLE
create a network classification class

### PR DESCRIPTION
**Pull request by BoltzmannBrain**

### CODE REVIEW

**Comment by BoltzmannBrain** [comment]
@marionleborgne please review. classification_network.py is the classification network factory we want; calling it returns a Network instance. classify_active_cells.py is an example; I left the majority of this script as is because I know it's where you're doing your experimentation.

**Comment by BoltzmannBrain** [comment]
@scottpurdy

File: classification/classification_network.py [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+  
+  # Link the sensor to the classifier to send in category labels.
+  # TODO: this link is useless right now because the CLAclassifier region
+  # compute() function doesn't work... we are currently feeding TM states and
+  # categories manually to the classifier via the customCompute() function.
+  network.link("sensor", "classifier", "UniformLink", "",
+      srcOutput = "categoryOut", destInput = "categoryIn")
+
+
+def createNetwork(args):
```
Comment (scottpurdy) [inline classification/classification_network.py]: 3 blank lines before top level functions (similar to classes)

**Comment by scottpurdy** [comment]
:+1: - One thing to keep an eye on is if it gets tedious having the region creation separate from the linking. I find it easier to add input links right after I add the region, interleaving the region and link creation. I don't know that my way is better though so I wouldn't change this unless it becomes tedious.

**Comment by BoltzmannBrain** [comment]
@scottpurdy thanks for the review. Easier, yes, but I think keeping region creation and linking separate forces one to be more mindful of what gets linked where, and will be better for unit testing. If it becomes tedious I can switch it back.

**Pull request merged**